storey/flow.py

```diff
@@ -82,7 +84,17 @@ async def _do(self, event):
             await self._default._do(event)
 
 
-Event = collections.namedtuple('Event', 'element key time awaitable_result')
+class Event:
+    def __init__(self, body, id=None, key=None, time=None, headers=None, method=None, path='/', content_type=None, awaitable_result=None):
+        self.body = body
+        self.id = id or uuid.uuid4().hex
+        self.key = key
+        self.time = time or datetime.now(timezone.utc)
```

Comment:
convert `time` to utc

Reply:
It really doesn't have to be UTC. It's up to the user.